### PR TITLE
config docs update for dynamic loading

### PR DESCRIPTION
**Pull request by rebstar6**

### CODE REVIEW

File: modules/docs/src/main/tut/operator/config-api.md [hunk 1]
```diff
@@ -101,105 +100,113 @@ vinyldns {
...
-      provisionedReads = 30
-      provisionedWrites = 30
+  # this list should include only the datastores being used by your instance
+  data-stores = ["mysql", "dynamodb"]
+  
+  dynamodb {
+      
+    # this is the path to the DynamoDB provider. This should not be edited
+    # from the default in reference.conf
+    class-name = "vinyldns.api.repository.mysql.MySqlDataStoreProvider"
```
Comment (pauljamescleary) [inline modules/docs/src/main/tut/operator/config-api.md]: heh, NOT the `MySqlDataStoreProvider`

File: modules/docs/src/main/tut/operator/config-api.md [hunk 2]
```diff
@@ -373,84 +390,78 @@ vinyldns {
...
-      user = "root"
-      password = "pass"
-      poolMaxSize = 20
-      connectionTimeoutMillis = 1000
-      maxLifeTime = 600000
+  # both datastore options are in use
+  data-stores = ["mysql", "dynamodb"]
+  
+  dynamodb {
+    class-name = "vinyldns.api.repository.mysql.MySqlDataStoreProvider"
```
Comment (pauljamescleary) [inline modules/docs/src/main/tut/operator/config-api.md]: and here

File: modules/docs/src/main/tut/operator/config-api.md (hunk 1 above)
Comment (rebstar6) [reply modules/docs/src/main/tut/operator/config-api.md]: lol so picky

**Comment by mitruly** [comment]
Think I'm good on this after updating `class-path` for `dyanmodb` as well.

**Pull request merged**